vars/kibanaPipeline.groovy

```diff
@@ -225,27 +228,33 @@ def sendKibanaMail() {
   }
 }
 
-def bash(script) {
-  sh "#!/bin/bash\n${script}"
+def bash(script, label="") {
```

Comment:
Do you think we could force definition of the label? Maybe same with runbld?

Reply:
Forced